src/backend/events.js

```diff
@@ -0,0 +1,28 @@
+import { stringify } from '../util'
+
+export function initEventsBackend (bridge, instanceMap, getInstanceName) {
+  bridge.on('trigger-event', (event) => {
+    const instance = instanceMap.get(event.instanceId)
+    instance._events[event.eventName][0](event.eventData)
+  })
+  instanceMap.forEach((instance) => {
+    const listeners = instance._events
+    if (listeners) {
+      Object.keys(listeners).forEach((eventName) => {
+        if (!eventName.startsWith('hook:')) {
```

Comment:
@yyx990803 I don't know if this is a valid approach to hook into the vue event system. I don't know too much about the internals of vue that I can say if this can have any side effects or not.